### ISSUE TITLE
selecting library on new install trigger 

### ISSUE THREAD

**Issue by spdegabrielle** [Metaxal/quickscript#17]
Hi, 

this error happens on a fresh install after DRR restart if you haven't run `(require quickscript-extra/register)`

```
directory-list: could not open directory
  path: /Users/spdegabrielle/Library/Preferences/quickscript/user-scripts
  system error: No such file or directory; errno=2
  context...:
   /Applications/Racket v7.1/collects/racket/private/kw-file.rkt:137:4
   /Users/spdegabrielle/Library/Racket/7.1/pkgs/quickscript/library-gui.rkt:44:4: set-files-lb
   /Users/spdegabrielle/Library/Racket/7.1/pkgs/quickscript/library-gui.rkt:156:4: dir-lb-select16
   /Users/spdegabrielle/Library/Racket/7.1/pkgs/quickscript/library-gui.rkt:27:0: make-library-gui6
   /Applications/Racket v7.1/share/pkgs/gui-lib/mred/private/mrmenu.rkt:250:14: command method in basic-selectable-menu-item%
   /Applications/Racket v7.1/collects/racket/private/more-scheme.rkt:148:2: call-with-break-parameterization
   /Applications/Racket v7.1/collects/racket/private/more-scheme.rkt:265:2: call-with-exception-handler
   /Applications/Racket v7.1/share/pkgs/gui-lib/mred/private/wx/common/queue.rkt:430:6
   /Applications/Racket v7.1/share/pkgs/gui-lib/mred/private/wx/common/queue.rkt:481:32
   /Applications/Racket v7.1/share/pkgs/gui-lib/mred/private/wx/common/queue.rkt:629:3
```

**Comment by spdegabrielle** [comment]
steps to reproduce
1. clean install of DrRacket
2. install quickscript
3. install quickscript-extra
4. restart DrRacket
5. select menu: Scripts>Manage Scripts>Library
6. error displayed

To resolve, type `(require quickscript-extra/register)` into the interactions/repl frame.

**Comment by Metaxal** [comment]
Thanks for the report!

I'll take a look soon. At least a better error message should be possible.

**Comment by Metaxal** [comment]
This should be fixed by 36e8a9f310b149996dd67b3cb65e3f8b832546b7
I'm closing this issue but don't hesitate to open a new one if it happens again.

Thanks for the report!
